keras_spiking/layers.py

```diff
@@ -846,7 +906,7 @@ def call_inference(self, inputs, states):
         #     AB = sy.Matrix([[-2/tau, -1/tau**2, 1/tau**2], [1, 0, 0], [0, 0, 0]])
         #     dAB = sy.exp(dt * AB).simplify()
         #     dA, dB = dAB[:2, :2], dAB[:2, 2:]
-        tau = self.tau_var[:, None, None] + 1e-8
+        tau = tf.maximum(self.tau[:, None, None], 1e-8)
```

Comment:
Not sure if we care about supporting float16, but the smallest value that float16 has is 6e-8. We could have an internal `eps` variable that depends on the layer's dtype.

Reply:
I'm not sure if we test with float16 at all, though, so maybe we'll need a larger task for supporting float16.

Reply:
Sorry merged before I saw this. But yeah I think supporting float16 would be a different (but worthwhile) enough thing to think about, so I'll make a new story for that.